File: games/bounce-evasion/bots/java/src/core/api/InitialData.java
package core.api;

import core.MatchDetails;

public class InitialData {
    public int __uid;
    public MatchDetails __matchDetails;
    public int mapWidth;
    public int mapHeight;
    public int sawSpawnDelay;
    public boolean[][] map;
    public Unit yourUnit;
    public Unit opponentUnit;
    public Coin[] coins;
    public Saw[] saws;
}

File: games/bounce-evasion/bots/kotlin/src/core/api/InitialData.kt
package core.api

import core.MatchDetails

class InitialData(
        var __uid: Int,
        var __matchDetails: MatchDetails,
        var sawSpawnDelay: Int,
        var mapWidth: Int,
        var mapHeight: Int,
        var map: Array<BooleanArray>,
        var boxSize: Int,
        var yourUnit: Unit,
        var opponentUnit: Unit,
        var saws: Array<Saw>
)
